scrooge-generator/src/main/resources/androidgen/struct_inner.mustache

```diff
@@ -85,7 +85,7 @@ public {{#is_final}}final {{/is_final}}{{#in_class}}static {{/in_class}}class {{
     return new {{name}}(this);
   }
 
-  @Override
+  @java.lang.Override
```

Comment:
Sorry if I'm misunderstanding (or wrong :)), but isn't `_root_` only supported in Scala?